src/drivers/mct/cime_config/config_component.xml

```diff
@@ -506,12 +506,12 @@
   <entry id="DOUT_S_SAVE_INTERIM_RESTART_FILES">
```

Comment:
Maybe we should move this to the config_component_acme.xml file so that cesm can maintain the original default value of FALSE?